Resources/Locale/en-US/alert-levels/alert-levels.ftl

```diff
@@ -28,8 +28,8 @@ alert-level-gamma-announcement = Central Command has ordered the Gamma security
 alert-level-gamma-instructions = All civilians are to immediately seek their nearest head for transportation to a secure location.
 
 alert-level-delta = Delta
-alert-level-delta-announcement = The station is currently under threat of imminent destruction. Crewmembers are advised to listen to heads of staff for more information.
-alert-level-delta-instructions = Crewmembers are advised to listen to heads of staff for more information.
+alert-level-delta-announcement = The nuclear charge has been activated. All staff is to deactivate the nuclear charge or evacuate immediately.
+alert-level-delta-instructions = Defuse the nuke or evacuate.
```

Comment:
code delta isn't used for just the nuke, it can be used by admins for events, this just makes it exclusively the nuke alert level